users/main.py

```diff
@@ -0,0 +1,22 @@
+import databases
```

Comment:
Isn't better "app.py" for this? [but I don't know the conventions...]

Reply:
the module inside it already exposes `app` so it might be a bit repetitive